kvdb-rocksdb/Cargo.toml

```diff
@@ -1,19 +1,22 @@
 [package]
 name = "kvdb-rocksdb"
-version = "0.1.0"
+version = "0.1.3"
 authors = ["Parity Technologies <admin@parity.io>"]
+repository = "https://github.com/paritytech/parity-common"
+description = "kvdb implementation backed by rocksDB"
+license = "GPL-3.0"
 
 [dependencies]
 elastic-array = "0.10"
-ethereum-types = "0.3"
 fs-swap = "0.2.1"
 interleaved-ordered = "0.1.0"
-kvdb = { path = "../kvdb" }
+kvdb = { version = "0.1", path = "../kvdb" }
 log = "0.3"
 num_cpus = "1.0"
 parking_lot = "0.6"
 regex = "0.2"
-rocksdb = { git = "https://github.com/paritytech/rust-rocksdb" }
+parity-rocksdb = "0.5.0"
 
 [dev-dependencies]
 tempdir = "0.3"
+ethereum-types = "0.4"
```

Comment:
My editor would complain about missing newline at end of file, but no strong opinion on it :P